org.gramps_project.Gramps.yml

```
@@ -170,14 +170,13 @@ modules:
         sha256:  ddec11449f37b5dffb4bca134d024623897c6140af1f9981a8acc512dbf6a7a5
 
     # Gramps does not see geocodeglib in platform, needed for place coordinate addon
-    # appears to not have versioned releases anymore, last git edit as of 202110 was from 202109
-    # geocodeglib appears to have changed prerequisites and needs to be commented out for now to allow Gramps to compile
+    # appears to not have versioned releases anymore, last git edit as of 202205 was from 202205
   - name: geocodeglibDependency
     buildsystem: meson
     sources:
       - type: archive
         url:  https://gitlab.gnome.org/GNOME/geocode-glib/-/archive/master/geocode-glib-master.tar.gz
-        sha256:  d2cfc8b096f112ad2446278e19473d316a048ed457f6349a05e68cc128a57f2d
+        sha256:  0d4c1bd9997dc0982e4e0a884067e4c4a1675d7cb51dd8205aba74dd02ed0342
```

Why not just use git source and pin it to specific commit? Then you won't need to update the checksum every time.
```suggestion
      - type: git
        url:  https://gitlab.gnome.org/GNOME/geocode-glib.git
        commit:  e5905e15b8192570a5a4b16c91fa86178fabb18f
```

---

Thank you for the idea.  While there are a lot of old dependencies in this flatpak for various reasons (usually to allow a popular add-on that has not been updated in a long time so it still requires old dependencies, or because the dependency won't compile in a flatpak, or in one case out of copyright issues), when a dependency can be updated, I try to update them.  That is why I made notes on each module, so that I can go back someday when I have time and see which dependencies can be updated, and when they were last checked for updates.  This is actually one of the security complaints with flatpaks, that we don't keep the dependencies up to date often enough.

So in the case of geocode-glib, unless they start pushing random updates that break the gramps flatpak or the frequency of their updates makes the maintenance of gramps flatpak too time-consuming, I think we should just keep the geocode-glib dependency up to date after verifying each time that the update doesn't break the gramps flatpak.

---

FYI updates can be automated with [flatpak-external-data-checker](https://github.com/flathub/flatpak-external-data-checker).